tests/ActivityloggerTest.php

```diff
@@ -243,4 +244,59 @@ public function it_returns_an_instance_of_the_activity_log_after_logging_when_us
 
         $this->assertInstanceOf($activityClassName, $activityModel);
     }
+
+    /** @test */
+    public function it_uses_any_guards_user_used()
```

Comment:
The name of this test can be improved. Change this to `it_uses_the_right_guard_when_determining_the_causer` (or something better)

This test feels too long to me. Could you break this up? You could opt to create a new `ActivityCauserTest` file.